projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -451,7 +403,16 @@ const LineChartInternal = forwardRef< SingleChartRef, LineChartProps >(
 												to={ providerTheme.backgroundColor }
 												{ ...seriesData.options?.gradient }
 												data-testid="line-gradient"
-											/>
+											>
+												{ seriesData.options?.gradient?.stops?.map( ( stop, stopIndex ) => (
+													<stop
+														key={ stopIndex }
+														offset={ stop.offset }
+														stopColor={ stop.color }
+														stopOpacity={ stop.opacity ?? 1 }
+													/>
+												) ) }
```

Comment:
Gives precise control over the gradient, so that we can stop it extending all the way to the x-axis

![Image](https://github.com/user-attachments/assets/0b550876-5936-42b4-b510-4c95635450c3)